### PR TITLE
Fix several sample code errors

### PR DESCRIPTION
**Pull request by 169**

### CODE REVIEW

File: docs/docs/expression_language/why.ipynb [hunk 1]
```diff
@@ -234,7 +234,7 @@
     "\n",
     "def batch_chain(topics: list) -> list:\n",
     "    with ThreadPoolExecutor(max_workers=5) as executor:\n",
-    "        return list(executor.map(invoke_chain, topics))\n",
+    "        return list(executor.map(chain.invoke, topics))\n",
```
Comment (sepiatone) [inline docs/docs/expression_language/why.ipynb:237]: The LCEL examples use the function invoke_chain() so this change is not correct

File: docs/docs/expression_language/why.ipynb [hunk 2]
```diff
@@ -863,7 +863,7 @@
     "\n",
     "def batch_chain(topics: list) -> list:\n",
     "    with ThreadPoolExecutor(max_workers=5) as executor:\n",
-    "        return list(executor.map(invoke_chain, topics))\n",
+    "        return list(executor.map(invoke_chain.invoke, topics))\n",
```
Comment (sepiatone) [inline docs/docs/expression_language/why.ipynb:866]: (Same as above) The LCEL examples use the function invoke_chain() so this change is not correct

**Review by sepiatone — CHANGES_REQUESTED** [review]

File: docs/docs/expression_language/why.ipynb (hunk 1 above)
Comment (169) [reply docs/docs/expression_language/why.ipynb]: Oh, yes, thanks

File: docs/docs/expression_language/why.ipynb (hunk 2 above)
Comment (169) [reply docs/docs/expression_language/why.ipynb]: Fixed: https://github.com/langchain-ai/langchain/pull/19382/commits/053b43e0d56a14df2fbb70ca9c5d4ce7fef97833

**Review by baskaryan — APPROVED** [review]

**Pull request merged**